cabal-install/cabal-install.cabal

```diff
@@ -27,7 +27,35 @@ Extra-Source-Files:
   README.md bash-completion/cabal bootstrap.sh changelog
 
   -- Generated with '../Cabal/misc/gen-extra-source-files.sh | sort'
-  tests/PackageTests/Freeze/my.cabal
+  tests/IntegrationTests/exec/common.sh
+  tests/IntegrationTests/exec/should_run/Foo.hs
+  tests/IntegrationTests/exec/should_run/My.hs
+  tests/IntegrationTests/exec/should_run/adds_sandbox_bin_directory_to_path.sh
```

Comment:
The "prints error message if given no argument" test for `exec` is missing, is this due to the Windows `.exe` problem?